src/org/omegat/core/dictionaries/LingvoDSL.java

```diff
@@ -172,6 +180,9 @@ public void finish() {
          */
         @Override
         public void visit(final DslArticle.Tag tag) {
+            if (inDetails && Preferences.isPreferenceDefault(Preferences.DICTIONARY_CONDENSED_VIEW, false)) {
```

Comment:
It seems better to be get preference in constructor for consistency among processing.